### PR TITLE
Update Tag Link to Route to Proper Admin Page

### PR DESCRIPTION
**Pull request by juliannatetreault**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
While working with tags, I discovered a broken link at the bottom of the `/t/<tag>/edit` page. This PR updates the link to route to the proper page, rather than an error page by adding `/edit` to the end of the link. Additionally, I have updated the edit link to read `[tag]'s admin page` rather than `[tag] admin` (see screenshot below).

## Related Tickets & Documents
N/A

## QA Instructions, Screenshots, Recordings
**BEFORE**:
- Navigate to `/t/<tag of your choice>/edit` and click the "`<tag name>` admin" link at the bottom of the page and run into a routing error:
Screen Shot 2021-01-22 at 4 00 23 PM
Screen Shot 2021-01-22 at 4 01 35 PM

**AFTER**:
- Navigate to `/t/<tag of your choice>/edit` and click the "`<tag name>` admin" link at the bottom of the page and redirect to the tag's admin edit page as expected:
Screen Shot 2021-01-22 at 4 00 23 PM
Screen Shot 2021-01-22 at 4 00 58 PM

- With newly updated link copy:
Screen Shot 2021-01-22 at 4 25 35 PM

### UI accessibility concerns?
N/A

## Added tests?

- [ ] Yes
- [x] No, and this is why: there is already a request spec, `tags_spec.rb`, that tests that tags are updated properly, etc.
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

Click Here

### CODE REVIEW

File: app/views/tags/edit.html.erb [hunk 1]
```diff
@@ -84,7 +84,7 @@
 
 <% if current_user.has_role?(:super_admin) || current_user.has_role?(:admin) %>
   <center>
-    <h1><a href="/admin/tags/<%= @tag.id %>" data-no-instant><%= @tag.name %> admin</a></h1>
+    <h1><a href="/admin/tags/<%= @tag.id %>/edit" data-no-instant><%= @tag.name %> admin</a></h1>
```
Comment (juliannatetreault) [inline app/views/tags/edit.html.erb:87]: **_Nit-pick_**: I feel like this link could be slightly more descriptive -- how do folks feel about updating this to read "<tag name> admin page" instead (or something more descriptive)?

**BEFORE PROPOSED CHANGE:**
Screen Shot 2021-01-22 at 4 03 01 PM

**AFTER PROPOSED CHANGE:**
Screen Shot 2021-01-22 at 4 04 11 PM

Comment (vaidehijoshi) [reply app/views/tags/edit.html.erb:87]: This is exactly the kind of nit-pick that I like -- agree with you completely that this isn't very clear. Either option is an improvement, but I like `[tag]'s admin page` the best as it seems the most clear to me! 🚀

**Review by vaidehijoshi — APPROVED** [review]
If I had to guess, I'd say that probably no one has ever clicked through to the admin view from the tag's edit page, which is why this broken link has been around but wasn't caught before 😰 Thank you for fixing this!

Approving this, and preemptively approving whichever link text option you end up changing to as well 👍🏽

File: app/views/tags/edit.html.erb (hunk 1 above)
Comment (juliannatetreault) [reply app/views/tags/edit.html.erb]: Awesome! I've updated the link to read `[tag]'s admin page`, @vaidehijoshi . 😄

**Review by benhalpern — APPROVED** [review]

**Review by rhymes — APPROVED** [review]

**Pull request merged**